scout/server/blueprints/public/views.py

```diff
@@ -21,7 +25,11 @@ def index():
     form = None
     if current_app.config.get("LDAP_HOST"):
         form = LDAPLoginForm()
-    return render_template("public/index.html", version=__version__, form=form)
+
+    badge_name = current_app.config.get("ACCREDITATION_BADGE")
```

Comment:
Check if badge image exists (here or elsewhere) and give reasonable warning, and either add placeholder text or hide the image element html.